docs/specs/clients/notify/authentication.md

```diff
@@ -118,8 +118,243 @@ A non-ideal way to avoid the race condition is for the sender to set the message
 
 ## wc_notifyDelete response
 
-- act - `notify_delete_response``
+- act - `notify_delete_response`
 - iss - did:key of dapp authentication key
 - aud - did:key of client identity key
 - app - did:web of app domain that this request is associated with 
   - Example: `did:web:app.example.com`
+
+## `wc_notifyGetNotifications`
+
+Paginated list of notifications with the most recent first. Unread notifications may be dispersed throughout the list and on subsequent pages.
```

Comment:
> Unread notifications may be dispersed throughout the list and on subsequent pages.
What do you mean here? Will there be difference between read or unread when it comes to pagination?

Reply:
Just saying the order is by timestamp, and unread notifications are not prioritized in the order so there technically could be 10 pages of read notifications and then 1 unread notification at the end